projects/04-training-api/examples/nokogiri_example.rb

```diff
@@ -0,0 +1,8 @@
+require 'nokogiri'
+# XPath: https://devhints.io/xpath
+# Nokogiri: https://nokogiri.org/#parsing-and-querying
+
+require 'open-uri' # Load content from URL
+
+doc = Nokogiri::HTML(open('https://12ft.io/'))
+doc.xpath('//meta[@property="og:url"]').first['content']
```

Comment:
Using Nokogiri implies knowledge of XML. If we go for this, note that we might also have to explain XML.

It shouldn't be something hard, but we'll introduce one more concept (or two if we count Nokogiri), and I think GraphQL has quite a few already.

That aside, they might get confused by Nokogiri syntax.

Reply:
Improved the example. We can answer questions around it, but that part should not be a blocker IMO